scripts/bundleSize/bundleSize.test.js

```diff
@@ -124,20 +108,6 @@ describe('bundleSize', () => {
       expect(didThrow).toBe(false);
     });
 
-    it('should use ora to show loading and success states', async () => {
```

Comment:
Hmm I could maybe consider redoing these tests, and checking that the correct message was logged to the console...? 🤔

Reply:
I've decided this is not required for now.